2.4/test/run

```diff
@@ -141,6 +141,23 @@ function run_s2i_test() {
   CONTAINER_ARGS='--user 1000' IMAGE_NAME=${IMAGE_NAME}-testapp ct_create_container testing-app-s2i
   cip=$(ct_get_cip 'testing-app-s2i')
   run "ct_test_response '${cip}:8080' 200 'This is a sample s2i application with static content.'"
+
+  # Let's see whether the automatically generated certificate works as expected
+  run "curl -k https://${cip}:8443 >output_generated_ssl_cert"
+  run "fgrep -e 'This is a sample s2i application with static content.' output_generated_ssl_cert"
+
+  # We also need to make sure the certificate is generated no sooner than in assemble phase,
+  # because shipping the same certs in the image would make it easy to exploit
+  # Let's see how old the certificate is and compare with how old the image is
+  image_age_s=$(ct_get_image_age_s "${IMAGE_NAME}")
```

Comment:
Wouldn't it make sense here to create a separate test case for checking certificate age?

Reply:
The reason to put it here was to not need to build a new application image again, which would be the same as the default one. I didn't want to introduce any assumptions that e.g. one test would depend on an application image built from previous test being available. On the other hand, building a new image should not take too much time, so we can likely separate it into a new test.